### DISCUSSION TITLE
PHPStorm chat panel always displaying "Sign in to Github"

### DISCUSSION THREAD

**Discussion by samuelgfeller** [community/community#114378] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Since a few days / weeks this happened regularly. When I open PHPStorm after starting the PC, I'm logged out of Github Copilot. I press "sign in" on the copilot icon in the navigation bar (highlighted in yellow in the screenshot) and then I'm immediately signed in but the chat window didn't get that info. I keep pressing on the sign in button but it does nothing. I cant use chat until I exit the IDE completely and reopen it. It's a minor inconvenience but happened often enough for it to bother me really. 

![image](https://github.com/community/community/assets/31797204/e4c0e43c-0400-4e6c-b885-f6b4e8130c7c)

**Comment by devashish2024** [comment]
It will if your connection is weak/cannot connect to copilot. However it works for me after signing in

**Comment by devashish2024** [comment]
Ok so probably a bug, try reinstalling phpstorm

**Reply by samuelgfeller** [reply]
I will try that next time it happens
